### PR TITLE
oSPARC services + files

### PR DESCRIPTION
**Pull request by ignapas**
# Description

* Fetches oSPARC service information from ``portal_api/sim/service`` and file associations from ``/portal_api/sim/file`` (needs new portal_api deployment)
* Lists oSPARC services in Tools & Resources / oSPARC services
* Uses information provided in file browser to show oSPARC viewers
* Makes it possible to instantiate new services in oSPARC by clicking in the button of each service search result
* Find data -> Data & Models in main navigation menu

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Manually

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have utilized components from the Design System Library where applicable
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: components/Resources/ServicesSearchResults.vue [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  },
+
+  methods: {
+    highlightMatches
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/_variables.scss';
```
Comment (egauzens) [inline components/Resources/ServicesSearchResults.vue:64]: Replace this with import with @import '@nih-sparc/sparc-design-system-components/src/assets/_variables.scss' and update variables accordingly (it looks like you should only have to update the colors using this for reference: https://nih-sparc.github.io/sparc-design-system-components/?path=/docs/components-colors--basic).

Also might as well do the same for ResourcesSearchResults.vue

i.e. $median -> $purple
      $light-grey -> $lineColor2
      $dark-sky -> $grey

**Review by egauzens — CHANGES_REQUESTED** [review]
Changes work well, but we may want to hide behind a flag until we are sure they are to be deployed to prod on Thursday

File: pages/resources/osparc-services/index.vue [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+import Breadcrumb from '@/components/Breadcrumb/Breadcrumb.vue'
+import SearchControlsContentful from '@/components/SearchControlsContentful/SearchControlsContentful.vue'
+import SortMenu from '@/components/SortMenu/SortMenu.vue'
+import ServicesSearchResults from '@/components/Resources/ServicesSearchResults.vue'
+import ToolsAndResourcesFacetMenu from '@/components/FacetMenu/ToolsAndResourcesFacetMenu.vue'
+import { fetchResources, searchTypes, sortOptions } from '../utils.ts'
+import SubmitToolSection from '@/components/Resources/SubmitToolSection.vue'
+
+export default {
+  name: 'SoftwarePage',
```
Comment (egauzens) [inline pages/resources/osparc-services/index.vue:108]: update name to OsparcServicesPage

**Comment by mguidon** [comment]
@egauzens In case this gets merged before the promotion to prod which will happen April 6. These changes need to be excluded since the CCB will not be able to test it prior to the CCB meeting. (Because the relevant endpoint are not yet on osparc prod.)

File: pages/resources/osparc-services/index.vue [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+  @media (min-width: 50rem) {
+    font-size: .75rem;
+  }
+  @media (min-width: 64rem) {
+    font-size: 1.25rem;
+    font-weight: 600;
+    text-transform: none;
+  }
+  &:hover,
+  &:focus,
```
[the remark this answers is not in the document]
Comment (ignapas) [reply pages/resources/osparc-services/index.vue]: done

File: components/Resources/ServicesSearchResults.vue (hunk 1 above)
Comment (ignapas) [reply components/Resources/ServicesSearchResults.vue]: done (please check)

File: pages/resources/osparc-services/index.vue (hunk 2 above)
Comment (ignapas) [reply pages/resources/osparc-services/index.vue]: done

**Pull request merged**